### PR TITLE
Fixed dropdown closable

### PR DESCRIPTION
**Pull request by mykhailo-diakovych**
Added empty customer card
Cleaned TableCell

## Summary by CodeRabbit

- **New Features**
  - Customer Card component now supports adding a new customer with a dedicated UI when no customer data is present.

- **Enhancements**
  - Updated Page Title components to accept tooltip options for improved flexibility.
  - Improved visual differentiation in Dynamic Type elements based on customer-related conditions.

- **Bug Fixes**
  - Removed unnecessary clipboard functionalities from the Table Cell component.
  - Fixed Dropdown component to allow click events to propagate as expected.

- **Refactor**
  - Standardized property naming by renaming `additionalContentVariant` to `variant` in Additional Content stories.

- **Style**
  - Adjusted class bindings and grid layout within Dynamic Type elements for better visual coherence.

### CODE REVIEW

File: packages/@orchidui-vue/src/Elements/AdditionalContent/OcAdditionalContent.stories.js
Comment (kewcoder) [inline packages/@orchidui-vue/src/Elements/AdditionalContent/OcAdditionalContent.stories.js:1]: @mykhailo-diakovych please change in Dashboard, if you remove this props

File: packages/@orchidui-vue/src/DataDisplay/Table/OcTableCell.vue [hunk 1]
```diff
@@ -63,20 +63,6 @@ const variantClass = computed(() => ({
   [Variants.DATETIME]: "px-4",
   [Variants.EMPTY]: "px-4 min-w-[48px]",
 }));
-const isCopied = ref(false);
```
Comment (kewcoder) [inline packages/@orchidui-vue/src/DataDisplay/Table/OcTableCell.vue:66]: @mykhailo-diakovych are you sure removing isCopy, will not be a problem?

Comment (mykhailo-diakovych) [reply packages/@orchidui-vue/src/DataDisplay/Table/OcTableCell.vue:66]: Yes, cause we already had this logic in CopyTooltip which is using in TableCell

**Review by kewcoder — APPROVED** [review]
all good

**Pull request merged**